src/test/index.js

```diff
@@ -0,0 +1,37 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../index';
+
+// Define the expect assertion
+const { expect } = chai;
+
+// chai middleware
+chai.use(chaiHttp);
+
+describe('INDEX js', () => {
```

Comment:
`INDEX js` should be rename to `Index js` let make the first letter uppercase only

Reply:
Okay, I will change it. Thanks

Reply:
done